packages/auth/src/AccessPolicyBuilder.js

```diff
@@ -14,6 +14,10 @@ export class AccessPolicyBuilder {
     this.setupCacheInvalidation();
   }
 
+  resetCaches() {
```

Comment:
@avaek this will now need to be updated from dev and conflicts resolved, since I have merged https://github.com/beyondessential/tupaia/pull/1484